app/javascript/.eslintrc.js

```diff
@@ -23,6 +23,11 @@ module.exports = {
     ],
     'import/prefer-default-export': 'off',
     'no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
+    'jsx-a11y/label-has-associated-control': [2, {
```

Comment:
I think being consistent and explicit is a good thing. I would write 'error' instead of the number. Like how it is on the previous line.